src/test/java/se/sundsvall/snailmail/integration/citizen/CitizenIntegrationTest.java

```diff
@@ -55,29 +54,30 @@ void testGetCitizenReturnsEmpty_shouldThrowException() {
 
 		when(citizenMock.getCitizen(uuid.toString())).thenReturn(Optional.empty());
 
+		final var stringUuid = uuid.toString();
 		assertThatExceptionOfType(ThrowableProblem.class)
-			.isThrownBy(() -> citizenIntegration.getCitizen(uuid.toString()))
+			.isThrownBy(() -> citizenIntegration.getCitizen(stringUuid))
 			.satisfies(problem -> {
 				assertThat(problem.getStatus()).isEqualTo(Status.INTERNAL_SERVER_ERROR);
 				assertThat(problem.getDetail()).isEqualTo("Failed to fetch citizen data from Citizen API");
 			});
 	}
 
 	@Test
-	void getCitizenThrowingException() {
+	void getCitizensThrowingException() {
```

Comment:
Cannot see the difference between this case and getCitizenThrowingException?

Reply:
Leftover from when there was a batch method for fetching multiples. Method no longer needed and neither is this test :)